environment/src/test/resources/default-anonymization-rules.yaml

```diff
@@ -0,0 +1,10 @@
+- pattern: \b([A-Za-z0-9]|[A-Za-z0-9][A-Za-z0-9\-\._]*[A-Za-z0-9])@(([A-Za-z0-9]|[A-Za-z][A-Za-z0-9\-]*[A-Za-z0-9])\.)+([A-Za-z0-9]|[A-Za-z0-9][A-Za-z0-9\-]*[A-Za-z0-9])\b
+  replacement: "email@redacted.host"
+- pattern: \d{4}[^\w]\d{4}[^\w]\d{4}[^\w]\d{4}
+  replacement: "XXXX-XXXX-XXXX-XXXX"
+- pattern: \d{3}[^\w]\d{2}[^\w]\d{4}
+  replacement: "XXX-XX-XXXX"
+- pattern: FPW\:\s+[\w|\W].*
+  replacement: "FPW: [REDACTED]"
+- pattern: cdpHashedPassword=.*[']
+  replacement: "[CDP PWD ATTRS REDACTED]"
```

Comment:
we should test the same file we use in prod, and not a different one.

Reply:
now the objects are mocked as the tests are not about to parsing the files (as config read from application.yml)